pipeline_tools/create_analysis_json.py

```diff
@@ -1,11 +1,16 @@
 #!/usr/bin/env python
 
-import json
 import argparse
+import base64
+import codecs
+from copy import deepcopy
+from google.cloud import storage
+import json
 
 
 def create_analysis(analysis_id, metadata_file, input_bundles_string, reference_bundle,
-        run_type, method, schema_version, analysis_file_version, inputs_file, outputs_file, format_map):
+        run_type, method, schema_version, analysis_file_version, inputs, output_url_to_md5,
+        extension_to_format):
```

Comment:
Since we are changing from `format_map` to `extension_to_format ` here, we might have to change all the other places, e.g. format_map_example.json and those static inputs files in this repo accordingly.

Reply:
I've renamed the variables in the Python code to be a bit more explicit, but I'm not sure it's worth renaming the files. We pass the file name as a parameter to the script, so there is no need for it to be identical to the variable name.